Lombiq.LoginAsAnybody.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -0,0 +1,24 @@
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Models;
+using Lombiq.Tests.UI.Services;
+using OpenQA.Selenium;
+using Shouldly;
+using System.Threading.Tasks;
+
+namespace Lombiq.LoginAsAnybody.Tests.UI.Extensions;
+
+public static class TestCaseUITestContextExtensions
+{
+    public static async Task TestLoginAsAnybodyAsync(this UITestContext context, bool checkBuildLink = false)
+    {
+        await context.SignInDirectlyAndGoToDashboardAsync();
```

Comment:
Also, add a test where a user with a role that can see the admin area can't login as anybody (so they can't see the button first, but do check that they can't use the controller action too.)